xgcm/grid.py

```diff
@@ -704,7 +709,7 @@ def max(
             vector_partner,
         )
 
-    def _wrap_and_replace_coords(self, da, data_new, position_to):
+    def _wrap_and_replace_coords(self, da, data_new, position_to, keep_coords):
```

Comment:
Could you set a default right here (e.g. `keep_coords=False`), that could help with all the tests below.